website/docs/r/automation_schedule.html.markdown

```diff
@@ -61,6 +65,20 @@ The following arguments are supported:
 
 * `timezone` - (Optional) The timezone of the start time. Defaults to `UTC`. For possible values see: https://msdn.microsoft.com/en-us/library/ms912391(v=winembedded.11).aspx
 
+* `week_days` - (Optional) List of days of the week that the job should execute on. Only valid for `Week`.
```

Comment:
Could we be a bit clearer by phrasing `Only valid for 'Week'` as `Only valid when frequency is 'Week'`? Applies to the new two properties to